src/pharmpy/modeling/plots.py

```diff
@@ -211,7 +211,13 @@ def plot_transformed_eta_distributions(
     return facet
 
 
-def plot_dv_vs_pred(model: Model, predictions: pd.DataFrame) -> alt.Chart:
+def plot_dv_vs_pred(
+    model: Model,
+    predictions: pd.DataFrame,
+    strat: str = None,
+    bins: int = 8,
+    uniform_scale: bool = True,
```

Comment:
I am a bit hesitant to this type of option. The problem is that it might be a slippery slope of options that customize the plot in different ways. The xposes have many options like this going in as `kwargs` type options to be applicable to many types of plots. One idea is to do changes to the plot object after plotting, but I don't know how difficult that is. What are the differences in the vega-lite json with and without `uniform_scale` ?